apps/console/src/public/deployment.config.json

```diff
@@ -112,7 +112,44 @@
                 "id": "QWNjb3VudERpc2FibGU"
             }
         ],
-        "outboundProvisioningConnectors": []
+        "outboundProvisioningConnectors": [],
+        "smsTemplates": [
```

Comment:
Hope we have added these to the default json (framework) for Identity Server.

Reply:
https://github.com/wso2/carbon-identity-framework/pull/6049

Reply:
Move `smsTemplates` also to the framework.